src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+import { useMemo } from 'react';
+import { Link, useParams } from 'react-router-dom';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const TabsPage = () => {
+  const { tabId = '' } = useParams();
+  const selectedTab = useMemo(
+    () => tabs.find(tab => tabId.includes(tab.id)),
+    [tabId],
+  );
+
+  return (
+    <>
```

Comment:
Use the container div here instead react fragment.
That's why we've wrapped all pages to this container